sdk/state/close.go

```diff
@@ -0,0 +1,120 @@
+package state
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/stellar/experimental-payment-channels/sdk/txbuild"
+	"github.com/stellar/go/txnbuild"
+	"github.com/stellar/go/xdr"
+)
+
+// The steps for a channel coordinated close are as followed:
+// 1. Initiator or Responder submits latest declaration tx
+// (in steps 2-4 Initiator and Responder are interchangeable, as long as they alternate)
+// 2. Initiator calls ProposeCoordinatedClose
```

Comment:
Why does the initiator have to call this step? Either participant should be able to take the first step.

Reply:
ya  it can be either, above step 2 in parentheses it says that. I'll move it down a line so that sticks out more

Reply:
https://github.com/stellar/experimental-payment-channels/pull/95/commits/b5083d0128fb1a48da68137f412f009fe198d3ee